pkg/model/gcemodel/api_loadbalancer.go

```diff
@@ -161,48 +161,49 @@ func createInternalLB(b *APILoadBalancerBuilder, c *fi.CloudupModelBuilderContex
 		InstanceGroupManagers: igms,
 	}
 	c.AddTask(bs)
-	for _, sn := range lbSpec.Subnets {
-		network, err := b.LinkToNetwork()
-		if err != nil {
-			return err
-		}
-		t := true
-		subnet := &gcetasks.Subnet{
-			Name:    s(sn.Name),
-			Network: network,
-			Shared:  &t,
-			// Override lifecycle because these subnets are specified
-			// to already exist.
-			Lifecycle: fi.LifecycleExistsAndWarnIfChanges,
-		}
-		// TODO: automatically associate forwarding rule to subnets if no subnets are specified here.
-		if subnetNotSpecified(sn, b.Cluster.Spec.Networking.Subnets) {
-			c.AddTask(subnet)
-		}
+
+	network, err := b.LinkToNetwork()
+	if err != nil {
+		return err
+	}
+
+	if len(b.Cluster.Spec.Networking.Subnets) != 1 {
```

Comment:
We can't support multiple subnets?  Is it that we would need multiple IPs?

Reply:
Not quite sure we support multiple subnets. I thought we don't, but I may be wrong.
If that's the case, then yes, we need multiple IPs.

Reply:
@justinsb Seems we don't support multiple subnets for GCE/GCP:
https://github.com/kubernetes/kops/blob/60a2e153a9590797e2e0a44efd3ea00c1589f0ad/upup/pkg/fi/cloudup/new_cluster.go#L643-L646

Reply:
I did something along the lines of supporting multiple subnets, but it's not pretty.